### ISSUE TITLE
Gamelog Pass-Through

### ISSUE THREAD

**Issue by JacobFischer** [siggame/Cerveau#7]
Currently the gamelog api is as follows:

host/gamelog/gameName/gameSession/[epoch]

While, that's cool, because theoretically someone could poke through gamelogs via session, that's not realistic. All the other siggame apps using this endpoint will have the identifier they need to grab a gamelog.

So instead move the gamelog api to:

host/gamelog/filename

where `filename` is the name of the file (without the .json.gz) in the output/gamelogs directory. Hell we might be able to just expose that folder and set the HTTP Response header's Content-Encoding: gzip so we just dump those bytes on them. That would be ideal.

Basically the current way is over-complicated for no reason.

**Comment by JacobFischer** [comment]
22c81fc was wrong issue... should probably refer to specific issues via repo/#...
